aim/web/ui/src/modules/core/engine/pipeline/index.ts

```diff
@@ -131,7 +131,7 @@ function createPipelineEngine<TStore, TObject>(
       const queryState = store.getState().query;
 
       if (!queryState.ranges.isInitial) {
-        const url = updateUrlSearchParam(
+        const url = getUpdatedUrl(
           'query',
           encode({
             ...queryState,
```

Comment:
why this need to spread ?

Reply:
there is only changed the name of the function